packages/contracts-bedrock/src/libraries/GasPayingToken.sol

```diff
@@ -53,9 +58,19 @@ library GasPayingToken {
     }
 
     /// @notice Writes the gas paying token, its decimals, name and symbol to the magic storage slot.
-    function set(address _token, uint8 _decimals, string memory _name, string memory _symbol) internal {
+    function set(address _token, uint8 _decimals, bytes32 _name, bytes32 _symbol) internal {
         Storage.setBytes32(GAS_PAYING_TOKEN_SLOT, bytes32(uint256(_decimals) << 160 | uint256(uint160(_token))));
-        Storage.setBytes32(GAS_PAYING_TOKEN_NAME_SLOT, bytes32(abi.encodePacked(_name)));
-        Storage.setBytes32(GAS_PAYING_TOKEN_SYMBOL_SLOT, bytes32(abi.encodePacked(_symbol)));
+        Storage.setBytes32(GAS_PAYING_TOKEN_NAME_SLOT, _name);
+        Storage.setBytes32(GAS_PAYING_TOKEN_SYMBOL_SLOT, _symbol);
+    }
+
+    /// @notice Maps a string to a bytes32 without leading or trailing zeroes.
+    function stringToBytes32Sanitized(string memory _str) internal pure returns (bytes32 _output) {
```

Comment:
smol nit: the types are inherent to the function signature so its not super necessary to put them in the function name itself. I think calling it `sanitize` could work or `toBytes32`. What do you think? Its just a really long fn name